src/routes/(admin)/tests/test-[type]/[id]/Configuration.svelte

```diff
@@ -209,6 +209,26 @@
 					</div>
 				</div>
 			{/if}
+			<div class="flex flex-row gap-3 align-top">
+				<div class="my-auto w-fit gap-4">
+					<Checkbox
```

Comment:
I feel the code is a bit restrictive here and assumes it will be only 2 languages. May be we should consider having a select where the user can select the desired language for the test.